src/OrchardCore.Modules/OrchardCore.OpenId/Controllers/AccessController.cs

```diff
@@ -99,7 +97,7 @@ public async Task<IActionResult> Authorize()
                     {
                         [OpenIddictServerAspNetCoreConstants.Properties.Error] = Errors.ConsentRequired,
                         [OpenIddictServerAspNetCoreConstants.Properties.ErrorDescription] =
-                            S["The logged in user is not allowed to access this client application."]
+                            "The logged in user is not allowed to access this client application."
```

Comment:
No more `S[]` ?

Reply:
It's deliberate. See https://github.com/OrchardCMS/OrchardCore/pull/7675/files#diff-44ca9283b8f821a762e5d0c72ad06292720058093eeb604fb872af0aee4f77a3R26-R27 for the explanation.